releasenotes-builder/config/checkstyle.properties

```diff
@@ -1,3 +1,5 @@
+# suppress inspection "UnusedProperty" for whole file
+# properties are used at "misc/build.xml"
```

Comment:
how comments change anything ?

Reply:
That is not a regular comment, that is a suppression comment.